lib/widgets/customized_textfieldbutton.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flutter/material.dart';
+
+class CustomizedTextField extends StatelessWidget {
+
+  final TextEditingController ? myController ;
+  final String ? hintText ;
+  final String ? helperText ;
+  final Color ? hintColor ;
+  final bool ? isPassword ;
```

Comment:
Burdaki değişkenler güzel. Login sayfasında email kısmında kullanmadığın değerler için de bir özellik yazmışsın. Mesela isPassword. Burda ?'nin anlamı bu değişkeni kullanmak zorunda değilim ama kullanabilirim. Ondan dolayı login sayfasında emaili yönetmek için oluşturduğun custom text field'de isPassword değişkenini silebilirsin.

Reply:
Dediğiniz gibi düzenlemeyi yaptım hocam.